compiler/one-cmds/one-build

```diff
@@ -3,6 +3,7 @@
 ''''export PY_PATH=${SCRIPT_PATH}/venv/bin/python                                       # '''
 ''''test -f ${PY_PATH} && exec ${PY_PATH} "$0" "$@"                                     # '''
 ''''echo "Error: Virtual environment not found. Please run 'one-prepare-venv' command." # '''
+''''exit 255                                                                            # '''
```

Comment:
[Q] What is semantic of `'''`?

Reply:
@mhs4670go Can you explain for us...?

Reply:
@chbahk It's a little trick to combine bash + python into one file. `''''` is just two `''`, which means nothing in bash. And, after bash calls `PY_PATH`, it will be `''' ... '''`, which means comments in python.

Reply:
Interesting :thinking:

Reply:
So I called https://linux.die.net/man/1/import in bash :smile: